virtualenvwrapper.el

```diff
@@ -226,6 +226,13 @@ prompting the user with the string PROMPT"
   (venv--set-venv-gud-pdb-command-name)
   (run-hooks 'venv-postactivate-hook))
 
+(defun venv--check-virtual-proj-path ()
```

Comment:
I think this function could have a better name. `virtual` is redundant, this whole project is about virtual environments. It also does more than check for a project path, it switches to it. I would call it `venv--switch-to-project-dir`.